dockerhub-script/README.md

```diff
@@ -0,0 +1,28 @@
+# Get DockerHub Token
+
+This composite action generates a DockerHub token and runs a script with it.
+
+## Inputs
+
+- `DOCKERHUB_USER` (required): DockerHub username
+- `DOCKERHUB_TOKEN` (required): DockerHub password
+- `script` (required): Script to run
+
+## Usage
+
+```yaml
+jobs:
+  example:
+    steps:
+      - name: Run With DockerHub Token
+        id: use_dockerhub_token
```

Comment:
```suggestion
```

Is there any reason `id` is included here?

`id` is typically used to reference outputs, but this composite job doesn't set any outputs.